src/Qowaiv.Validation.TestTools/FluentAssertions/ResultAssertionsBase.cs

```diff
@@ -0,0 +1,58 @@
+﻿using FluentAssertions.Execution;
+using Qowaiv.Validation.Abstractions;
+using System.Linq;
+using System.Text;
+
+namespace FluentAssertions.Qowaiv.Validation
+{
+    /// <summary>Contains a number of methods to assert the (in)validness of a <see cref="Result"/>.</summary>
+    public abstract class ResultAssertionsBase<TSubject>
+        where TSubject : Result
+    {
+        /// <summary>Creates a new instance of the <see cref="ResultActExtensions"/> class.</summary>
```

Comment:
Is this comment correct?

Reply:
I think so, do you have a better/other idea?